<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="write_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>97</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Записать</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Кредитор</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Заёмщик</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>181</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="le_lender"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_amount"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_borrower">
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="integrity_button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Проверить целостность</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_integrity">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
